index.html

```diff
@@ -919,8 +925,8 @@ <h4>parseDerivedProofValue</h4>
             </li>
             <li>
 Return <em>derived proof value</em> as an object with properties set to the five
-elements, using the names "`bbsProof`", "`labelMap`", "`mandatoryIndexes`", and
-"`selectiveIndexes`" respectively.
+elements, using the names "`bbsProof`", "`labelMap`", "`mandatoryIndexes`",
+"`selectiveIndexes`", and "`presentationHeader`" respectively.
```

Comment:
and now the question is whether they should be ```"`this way`"``` or ``` `this way` ```...

Either way, they should be consistent throughout.

Reply:
I'll take your recommendation. I've been down in the details of test vectors and such lately.